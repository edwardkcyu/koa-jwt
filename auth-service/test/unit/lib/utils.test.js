const { hash } = require('../../../src/lib/utils');

describe('utils.hash', () => {
  test('should correctly create SHA256 hash', () => {
    expect(hash('abcd1234password')).toMatchSnapshot();
  });
});
