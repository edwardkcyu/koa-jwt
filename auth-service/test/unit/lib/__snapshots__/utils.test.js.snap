// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils.hash should correctly create SHA256 hash 1`] = `"cba0021ca36477ed4be95626316ac66e31ef7d73e8f9e3888e36737e1a9ae597"`;
